data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -88,7 +88,9 @@ then
 		--manifest-image=${OPENSHIFT_HYPERKUBE_IMAGE} \
 		--asset-input-dir=/assets/tls \
 		--asset-output-dir=/assets/kube-scheduler-bootstrap \
-		--config-output-file=/assets/kube-scheduler-bootstrap/config
+		--config-output-file=/assets/kube-scheduler-bootstrap/config \
+		--config-override-files=/assets/bootkube-config-overrides/kube-scheduler-config-overrides.yaml \
```

Comment:
where is this coming from? the bootstrap override file is applied internally in the operator render command.